chainer/gradient_check.py

```diff
@@ -517,15 +517,15 @@ def _compare_gradients(self, gx_numeric, gx_backward, directions):
                 gx_numeric, gx_backward, atol=atol, rtol=rtol)
         except AssertionError as e:
             eps = self.eps
-            x_data = self.x_data
-            y_grad = self.y_grad
+            xs = self.xs
+            gys = self.gys
             f = six.StringIO()
             f.write('check_backward failed (eps={} atol={} rtol={})\n'.format(
                 eps, atol, rtol))
-            for i, x_ in enumerate(x_data):
+            for i, x_ in enumerate(xs):
```

Comment:
Do you need the underscore suffix? It's not introduced by this PR but maybe it could be fixed since the intension is to clean up the code. What do you think?

Reply:
Fixed.